### DISCUSSION TITLE
Help: Blocked

### DISCUSSION THREAD

**Discussion by Nyunnnk** [3kh0/ext-remover#767] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
The bookmark bar and entering scripts into the URL bar are blocked. Same thing with Inspect Element Anyway to bypass?

**Comment by doofenator** [comment, marked as answer]
ur kinda screwed, either that means javascript is blocked or you have to find a different way to run js, like ublock

**Reply by gooseisgoose** [reply]
yah i don't think theres any thing you can do. its the same situation here. but if you want to get a extension if you have microsoft edge you can go the the chrome wed store on edge and it will give you the option to install that extension on edge.
